ext/TrixiConvexECOSExt.jl

```diff
@@ -28,10 +29,14 @@ using Trixi: Trixi, undo_normalization!, bisect_stability_polynomial, @muladd
 # relative to consistency order.
 function Trixi.undo_normalization!(gamma_opt, consistency_order, num_stage_evals)
     for k in (consistency_order + 1):num_stage_evals
-        gamma_opt[k - consistency_order] = gamma_opt[k - consistency_order] /
-                                           factorial(k)
+        gamma_opt[k - consistency_order] /= factorial(k)
+    end
+end
+
+function Trixi.undo_normalization_PERK4!(gamma_opt, num_stage_evals)
+    for k in 1:(num_stage_evals - 5)
+        gamma_opt[k] /= factorial(k + 4)
```

Comment:
I guess we could replace `consistency_order` with something like `offset` and then re-use the standard method.